src/styles/card.scss

```diff
@@ -0,0 +1,101 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 18px;
+  color: $main-text-color;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
```

Comment:
This should be in `index.scss` or a separate `common.scss`, because these are general styles which impact everything